lib/CustomFields/components/CustomFieldsForm/CustomFieldsForm.js

```diff
@@ -156,7 +156,7 @@ const CustomFieldsForm = ({
         disabled={buttonsDisabled}
         data-test-custom-fields-save-button
       >
-        <FormattedMessage id="stripes-smart-components.customFields.save" />
+        <FormattedMessage id="stripes-smart-components.customFields.saveAndClose" />
```

Comment:
In fact we have `stripes-core.button.saveAndClose` for common labels like this, but the translation there is `Save and close` rather than `Save & close`. Your call.